src/memorial_logger.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#ifndef CATA_MEMORIAL_LOGGER_H
+#define CATA_MEMORIAL_LOGGER_H
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "event_bus.h"
+
+class memorial_logger : public event_subscriber
+{
+    public:
+        void clear();
+
+        void add_memorial_log( const std::string &/*male_msg*/,
+                               const std::string &/*female_msg*/ );
```

Comment:
Why are the parameter names commented out?

Reply:
Probably no good reason.  I just copied this from `player`.  Will change.